1.8/usage/service-discovery/load-balancing-vips/virtual-ip-addresses.md

```diff
@@ -4,6 +4,12 @@ menu_order: 10
 ---
 DC/OS can map traffic from a single Virtual IP (VIP) to multiple IP addresses and ports. DC/OS VIPs are name-based, which means clients connect with a service address instead of an IP address. DC/OS generates name-based VIPs such that the cannot collide with IP VIPs, so administrators don't need to carefully manage name-based VIPs to avoid collision. This also means that name-based VIPs can be automatically created when the service is installed.
 
+A named VIP contains 3 components:
+
+ * The private virtual IP address,
```

Comment:
* Private virtual IP address
* Service port (the port which the service is available on)
* Service name